File: StackSearch/StackSearch/Models/MasterModel.swift
//
//  MasterModel.swift
//  StackSearch
//
//  Created by Saiful I. Wicaksana on 11/08/18.
//  Copyright © 2018 icaksama. All rights reserved.
//

import Foundation

internal struct MasterModel: Codable {
    internal let items: [Item]
    internal let has_more: Bool
    internal let quota_max: Int
    internal let quota_remaining: Int
}

File: AStackSearch/app/src/main/java/com/icaksama/astacksearch/models/MasterModel.java
package com.icaksama.astacksearch.models;

import com.google.gson.annotations.SerializedName;
import com.icaksama.astacksearch.models.items.Item;

import java.util.List;

/**
 * Created by icaksama on 13/08/18.
 */

public class MasterModel {

    @SerializedName("items")
    public List<Item> items;

    @SerializedName("has_more")
    public boolean has_more;

    @SerializedName("quota_max")
    public int quota_max;

    @SerializedName("quota_remaining")
    public int quota_remaining;
}
